src/Uno.UI/UI/Xaml/Controls/PersonPicture/PersonPicture.cs

```diff
@@ -0,0 +1,699 @@
+using Windows.ApplicationModel.Contacts;
+using Windows.ApplicationModel.Resources;
+using Windows.Foundation;
+using Windows.Storage.Streams;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+using Windows.UI.Xaml.Shapes;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public  partial class PersonPicture : Controls.Control
+	{
+		/// <summary>
+		/// XAML Element for the first TextBlock matching x:Name of InitialsTextBlock.
+		/// </summary>
+		TextBlock m_initialsTextBlock;
+
+		/// <summary>
+		/// XAML Element for the first TextBlock matching x:Name of BadgeNumberTextBlock.
+		/// </summary>
+		TextBlock m_badgeNumberTextBlock;
+
+		/// <summary>
+		/// XAML Element for the first TextBlock matching x:Name of BadgeGlyphIcon.
+		/// </summary>
+		FontIcon m_badgeGlyphIcon;
+
+		/// <summary>
+		/// XAML Element for the first ImageBrush matching x:Name of BadgeImageBrush.
+		/// </summary>
+		ImageBrush m_badgeImageBrush;
+
+		/// <summary>
+		/// XAML Element for the first Ellipse matching x:Name of BadgingBackgroundEllipse.
+		/// </summary>
+		Ellipse m_badgingEllipse;
+
+		/// <summary>
+		/// XAML Element for the first Ellipse matching x:Name of BadgingEllipse.
+		/// </summary>
+		Ellipse m_badgingBackgroundEllipse;
+
+		/// <summary>
+		/// The async operation object representing the loading and assignment of the Thumbnail.
+		/// </summary>
+		IAsyncOperation<IRandomAccessStreamWithContentType> m_profilePictureReadAsync;
+
+		/// <summary>
+		/// The initials from the DisplayName property.
+		/// </summary>
+		string m_displayNameInitials;
+
+		/// <summary>
+		/// The initials from the Contact property.
+		/// </summary>
+		string m_contactDisplayNameInitials;
+
+		/// <summary>
+		/// The ImageSource from the Contact property.
+		/// </summary>
+		ImageSource m_contactImageSource;
+		private readonly string SR_GroupName				= "GroupName";
+		private readonly string SR_PersonName				= "PersonName";
+		private readonly string SR_BadgeItemTextOverride	= "BadgeItemTextOverride";
+		private readonly string SR_BadgeIconTextOverride	= "BadgeIconTextOverride";
+		private readonly string SR_BadgeIcon				= "BadgeIcon";
+		private readonly string SR_BadgeItemSingular		= "BadgeItemSingular";
+		private readonly string SR_BadgeItemPlural7			= "BadgeItemPlural7";
+		private readonly string SR_BadgeItemPlural2			= "BadgeItemPlural2";
+		private readonly string SR_BadgeItemPlural5			= "BadgeItemPlural5";
+		private readonly string SR_BadgeItemPlural6			= "BadgeItemPlural6";
+		private readonly string SR_BadgeItemPlural1			= "BadgeItemPlural1";
+		private readonly string SR_BadgeItemPlural3			= "BadgeItemPlural3";
+		private readonly string SR_BadgeItemPlural4			= "BadgeItemPlural4";
+
+		public PersonPictureTemplateSettings TemplateSettings { get; private set; }
+
+		public PersonPicture()
+		{
+			// DefaultStyleKey = typeof(PersonPicture);
```

Comment:
should be safe to set this, it'll just do nothing (yet)? Or am I missing something?

Reply:
True, adding it back